test/commands/attest/apply.test.ts

```diff
@@ -0,0 +1,55 @@
+import {expect, test} from '@oclif/test'
+import * as tmp from 'tmp'
+import path from 'path'
+import fs from 'fs'
+import {ExecJSON} from 'inspecjs'
+import _ from 'lodash'
+
+export function omitHDFChangingFields(
```

Comment:
we should probably put this (and similar functions that are currently just being duplicated like omitVersions) into a utils file like how we do in hdfconverters

Reply:
or maybe replace the other files' omitversions implementation with this one after it's moved into a utils file